packages/rolldown/src/options/input-options.ts

```diff
@@ -138,6 +138,19 @@ export interface InputOptions {
     viteMode?: boolean;
     resolveNewUrlToAsset?: boolean;
     hmr?: HmrOptions;
+    /**
+     * Attach debug information to the output bundle.
+     *
+     * - Type: `'none' | 'simple' | 'full'`
+     * - Default: `'simple'`
+     *
+     * - `none`: No debug information is attached.
+     * - `simple`: Attach comments indicating which files the bundled code comes from. These comments could be removed by the minifier.
+     * - `full`: Attach detailed debug information to the output bundle. These comments are using legal comment syntax, so they won't be removed by the minifier.
```

Comment:
Why not just use normal comments instead, so we don't need to worry about people misusing it accidentally and reporting bugs.

Reply:
Because we sometimes need to debug output with enabling minifer.